support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -2,11 +2,49 @@ import { onConsentChange } from '@guardian/consent-management-platform';
 import { loadScript } from '@guardian/libs';
 import type { Participations } from 'helpers/abTests/abtest';
 import { isSwitchOn } from 'helpers/globalsAndSwitches/globals';
+import type { IsoCurrency } from 'helpers/internationalisation/currency';
+import type { BillingPeriod } from 'helpers/productPrice/billingPeriods';
+import type { ProductPrice } from 'helpers/productPrice/productPrices';
 import { logException } from 'helpers/utilities/logger';
+import { getAnnualValue } from './quantumMetricHelpers';
 
-type SendEventABTestId = 30;
+type SendEventTestParticipationId = 30;
+type SendEventDigiSubCheckoutStart = 75;
+type SendEventPaperSubCheckoutStart = 76;
+type SendEventGuardianWeeklySubCheckoutStart = 77;
+type SendEventDigiSubGiftCheckoutStart = 78;
+type SendEventGuardianWeeklySubGiftCheckoutStart = 79;
+type SendEventDigiSubConversion = 31;
+type SendEventPaperSubConversion = 67;
+type SendEventGuardianWeeklySubConversion = 68;
+type SendEventDigiSubGiftConversion = 69;
+type SendEventGuardianWeeklySubGiftConversion = 70;
 
-export type SendEventId = SendEventABTestId;
+type SendEventCheckoutStartId =
+	| SendEventDigiSubCheckoutStart
+	| SendEventPaperSubCheckoutStart
+	| SendEventGuardianWeeklySubCheckoutStart
+	| SendEventDigiSubGiftCheckoutStart
+	| SendEventGuardianWeeklySubGiftCheckoutStart;
+
+type SendEventCheckoutConversionId =
+	| SendEventDigiSubConversion
+	| SendEventPaperSubConversion
+	| SendEventGuardianWeeklySubConversion
+	| SendEventDigiSubGiftConversion
+	| SendEventGuardianWeeklySubGiftConversion;
+
+type SendEventId =
+	| SendEventTestParticipationId
+	| SendEventCheckoutStartId
+	| SendEventCheckoutConversionId;
+
+const testParticipationId = 30;
+const digiSubCheckoutStartId = 75;
+const paperSubCheckoutStartId = 76;
+const guardianWeeklySubCheckoutStart = 77;
+const digiSubGiftCheckoutStartId = 78;
+const guardianWeeklySubGiftCheckoutStart = 79;
```

Comment:
that would remove the need for these extra `consts` which means there's just one less place for errors